apprentice_teacher.rb

```diff
@@ -1,22 +1,19 @@
-class ApprenticeTeacher
-  attr_reader :age, :salary, :phase, :target_raise
-  attr_accessor :name
+require_relative 'human'
+
+class ApprenticeTeacher < Teacher
+  include HighFiveable
+  include Payable
+
+  attr_reader :salary, :phase, :target_raise
```

Comment:
The other teacher has most of these as well.  Pull all attr_readers up to the highest level that makes sense. (i.e. the class where all subclasses of that class should have it).